src/go/k8s/pkg/resources/statefulset_update.go

```diff
@@ -35,8 +35,8 @@ import (
 const (
 	// RequeueDuration is the time controller should
 	// requeue resource reconciliation.
-	RequeueDuration = time.Second * 10
-	adminAPITimeout = time.Second * 2
+	RequeueDuration        = time.Second * 10
+	defaultAdminAPITimeout = time.Second * 2
```

Comment:
If we already know that this timeout is insufficient in some cases, would it be prudent to raise the default? Is there any harm in raising it?